### DISCUSSION TITLE
26 - Full Text Search for Publishing

### DISCUSSION THREAD

**Discussion by kevinslin** [dendronhq/dendron#1320] [RFCs/Ideas]
Details of rfc [here](https://wiki.dendron.so/notes/VaCraUDApW60xi0EDfBK9.html)

**Comment by SeriousBug** [comment]
On the use of the bar: I'm personally a fan of unified interfaces, and would prefer a combined bar. One thing we could do to simplify things is to have separate fulltext search and lookup modes.

This could be toggled manually with a switch, and/or automatically selected based on the query text. For example, if the user types `foo bar` then that would be a full text search, and if they type `foo.bar` then that would be a lookup. One drawback hovewer is that this is ambiguous when the user only types a single word.

This is similar to how the URL bar in a browser both allows direct navigation to URLs and also search.
